setup.py

```diff
@@ -37,11 +37,16 @@
 setup_requires = ["pytest-runner>=2.6.2"]
 
 install_requires = [
-    "github3.py>=0.9.3",
+    "black>=19.3b0",
```

Comment:
If the dependencies are related to tests, code checking, etc. they should be placed in `tests_require`. I think `black` though is more of a global binary, so no need to add it in dependencies